tests/test_sphere.py

```diff
@@ -48,7 +51,15 @@ class TestSphere(OptTest):
             "c2": 1.25,  # Social Parameter
             "stopCriteria": 0,  # 0: maxOuterIter, 1: convergence
             "seed": 1235,
-        }
+        },
+        "NSGA2": {
```

Comment:
Now we skip the NSGA2 test, should we remove the NSGA2 options here?

Reply:
Added comment, in the future I'd like to bring it back once #448 is addressed